validators/stake_nft_mint.ak

```diff
@@ -138,9 +141,6 @@ fn lock_transaction(
           stake_nft.2nd,
         )
 
-      //Only allow mint of 2 assets (reference and stake nft)
-      let no_extra_mint = list.length(minted) == 2
```

Comment:
Auditors note: This removal is safe because we `expect` this to be true on line 56